package.json

```diff
@@ -23,7 +24,7 @@
   },
   "homepage": "https://github.com/panoplyio/eslint-config-panoplioy#readme",
   "dependencies": {
-    "eslint-config-google": "^0.14.0",
```

Comment:
We should remove the style guide from our Confluence as well, I remember we refer to it.

Reply:
@OmriGLevy can't find any mention of this, you mean this doc? https://panoply.atlassian.net/wiki/spaces/RD/pages/1014858095/Coding+Style+Conventions